substrate/client/transaction-pool/api/src/lib.rs

```diff
@@ -247,6 +247,9 @@ pub trait TransactionPool: Send + Sync {
 	fn remove_invalid(&self, hashes: &[TxHash<Self>]) -> Vec<Arc<Self::InPoolTransaction>>;
 
 	// *** logging
+	/// Get futures transaction list.
+	fn futures(&self) -> Vec<Self::InPoolTransaction>;
```

Comment:
I'd be tempted to call this `future_transactions` to avoid any confusion with rust futures.